renderdoc/driver/gl/gl_renderstate.cpp

```diff
@@ -1205,7 +1206,7 @@ void GLRenderState::ApplyState(void *ctx, WrappedOpenGL *gl)
   if(HasExt[ARB_shader_image_load_store])
   {
     GLuint maxImages = 0;
-    m_Real->glGetIntegerv(eGL_MAX_IMAGE_UNITS, (GLint *)&maxImages);
+    m_Real->glGetIntegerv(eGL_MAX_TEXTURE_IMAGE_UNITS, (GLint *)&maxImages);
```

Comment:
What's the reason for this change? `GL_MAX_TEXTURE_IMAGE_UNITS` is (confusingly) the number of active units you can select with `glActiveTexture`. It seems to be true for ES3 and desktop GL.

`GL_MAX_IMAGE_UNITS` is the number of image load/store bindings, which I think is what we want here.